tools/cluster-up/cluster-up.sh

```diff
@@ -367,7 +367,7 @@ then
     do
         # The libvirt vagrant provider doesn't support boot_timeout, so we ignore
         # errors and check for ssh to come up later on.
```

Comment:
It looks like the ignoring the errors is due to boot_timeout not being supported for libvirt based hosts. Is there an error message being covered up by redirecting to /dev/null?

Reply:
Probably, but it was also covering up vagrant errors when bringing up the backend while I was developing another change. I'd rather be noisy but work than fail silently.

Reply:
Makes sense.